src/Umbraco.Core/Persistence/Migrations/Upgrades/TargetVersionSevenThreeZero/AddRelationTypeForDocumentOnDelete.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Umbraco.Core.Configuration;
+using Umbraco.Core.Models.Rdbms;
+
+namespace Umbraco.Core.Persistence.Migrations.Upgrades.TargetVersionSevenThreeZero
+{
+    [Migration("7.3.0", 0, GlobalSettings.UmbracoMigrationName)]
+    public class AddRelationTypeForDocumentOnDelete : MigrationBase
+    {
+        public override void Up()
+        {
+            Execute.Code(AddRelationType);
+        }
+
+        public static string AddRelationType(Database database)
+        {
+            database.Insert("umbracoRelationType", "id", false, new RelationTypeDto { Id = 2, Alias = Constants.Conventions.Relations.RelateParentDocumentOnDelete, ChildObjectType = new Guid(Constants.ObjectTypes.Document), ParentObjectType = new Guid(Constants.ObjectTypes.Document), Dual = false, Name = "Relate Parent Document On Delete" });
```

Comment:
You should probably not assume id = 2 here. TBH I'm not sure if its actually ignored for inserts to this table, but the migration will fail if someone has created a RelationType, which has the same id (2).
At least double check that this migration works with existing custom RelationTypes :smiley: